### PR TITLE
Google Maps API and Twitter API

### PR DESCRIPTION
**Pull request by mra1385**
Added Google Maps API and started Twitter API

### CODE REVIEW

File: myapp/LocalTweets.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+__author__ = 'Mike Azar'
+from config import API_Key, API_Secret
+
+pass
```
Comment (martinzugnoni) [inline myapp/LocalTweets.py]: Remove this "pass"

File: myapp/GMaps.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+__author__ = 'Mike Azar'
+
+from instance.config import Google_API
+
+def get_coordinates(StNum,StName,StType,City, State):
+    r = requests.get('https://maps.googleapis.com/maps/api/geocode/json?address=' + StNum + '+' + StName + '+' + StType + ',+' + City + ',+' + State + '&sensor=FALSE' + Google_API)
```
Comment (martinzugnoni) [inline myapp/GMaps.py]: I don't see the `requests` module import. Did you try if this is working?

File: myapp/GMaps.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+__author__ = 'Mike Azar'
+
+from instance.config import Google_API
+
+def get_coordinates(StNum,StName,StType,City, State):
```
Comment (martinzugnoni) [inline myapp/GMaps.py]: This is a perfect moment to implement a test case! :smile:

File: myapp/GMaps.py (hunk 2 above)
Comment (mra1385) [inline myapp/GMaps.py]: Ahh, I forgot to copy that line from my text editor. Thanks.

File: myapp/GMaps.py (hunk 3 above)
Comment (mra1385) [inline myapp/GMaps.py]: You mean a unit test?

File: myapp/LocalTweets.py (hunk 1 above)
Comment (mra1385) [inline myapp/LocalTweets.py]: Removed.

File: myapp/GMaps.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+__author__ = 'Mike Azar'
+
+from instance.config import Google_API
+import requests
+
+def get_coordinates(StNum,StName,StType,City, State):
```
Comment (santiagobasulto) [inline myapp/GMaps.py]: The recommended way  is to use underscores to split words: `st_num` instead `StNum`

Comment (mra1385) [inline myapp/GMaps.py]: Oh right. OK I'll modify it thanks.

**Comment by bemery94** [comment]
Hey Mike can you please add a new directory for all the API stuff? After that looks like we can merge it with main!

**Pull request merged**